cmd/vendor/github.com/cloudflare/cfssl/revoke/revoke.go

```diff
@@ -31,17 +31,20 @@ type Revoke struct {
 	// status of a certificate (i.e. due to network failure) causes
 	// verification to fail (a hard failure).
 	HardFail bool
+	// HardFailLck is a Mutex locker for the HardFail
+	HardFailLck sync.Mutex
```

Comment:
which rev in the cfssl repo has `type Revoke struct`? vendored deps shouldn't diverge from upstream...